lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -316,7 +317,7 @@ def unhide(unit):
         # scheduled chapter
         ccx = CustomCourseForEdX.objects.get()
         course_start = get_override_for_ccx(ccx, self.course, 'start')
-        self.assertEqual(str(course_start)[:-9], u'2014-11-20 00:00')
+        self.assertEqual(str(course_start)[:-9], str(self.chapters[0].start)[:-9])
```

Comment:
No tests for the `get_date` function or are they implicit in the lines you added?